crates/scope/lib.rs

```diff
@@ -101,10 +103,9 @@ impl Drop for ScopeGuard {
 pub fn logger() -> Logger {
     TL_SCOPES.with(|s| {
         let s = s.borrow();
-        if s.is_empty() {
-            (*GLOBAL_LOGGER.get()).clone()
-        } else {
-            s[s.len() - 1].clone()
+        match s.last() {
+          Some(logger) => logger.clone(),
```

Comment:
It's not a big deal, but let's stick with `rustfmt` mandated formatting (4 spaces etc.). I generally from time to time `rustfmt` everything, so not a big deal.